### PR TITLE
[DEV APPROVED] - Homepage enhancement

### PR DESCRIPTION
**Pull request by margOnline**
[TP 9261](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=userstory/9261)

This pr adds the final components to the homepage view - 2 large buttons, both have static content and links.

### CODE REVIEW

File: app/views/homepages/show.html.erb [hunk 1]
```diff
@@ -39,5 +39,14 @@
         <%= render 'shared/latest_news', latest_news: @latest_news %>
       </div>
     </div>
+
+    <div class="l-2col">
+      <div class="l-2col-even">
+        <%= render 'shared/large_cta', url: 'https://fincap.moneyadviceservice.org.uk/en/articles/our-approach', icon: 'document', text: 'I am interested in research and evaluation' %>
```
Comment (tomas-stefano) [inline app/views/homepages/show.html.erb]: Can we change to article_path('our-approach')?

File: app/views/homepages/show.html.erb [hunk 2]
```diff
@@ -39,5 +39,14 @@
...
+      </div>
+      <div class="l-2col-even">
+        <%= render 'shared/large_cta', url: 'https://moneyadviceservice.org.uk', icon: 'money', text: 'I need help to manage my money' %>
```
Comment (tomas-stefano) [inline app/views/homepages/show.html.erb]: Is this the right link?

Comment (margOnline) [reply app/views/homepages/show.html.erb]: @VenetiaC said it should go to 'MAS homepage'

Comment (tomas-stefano) [reply app/views/homepages/show.html.erb]: Ok 👍

**Comment by margOnline** [comment]
@tomas-stefano ♻️ please note that this will fail unless there is an article on the cms with the slug 'our approach'

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**